src/data/faq_de.json

```
@@ -224,6 +224,17 @@
                         ]
+                    },
+                    {
+                        "title": "Wie funktioniert der Papierkorb in der Corona-Warn-App?",
+                        "anchor": "cert_recycle_bin",
+                        "active": true,
+                        "textblock": [
+                            "Ab Version 2.13 werden Impf-, Genesenen- oder Testzertifikate, die Sie gelöscht haben, vorläufig in einen Papierkorb verschoben. Dies ermöglicht Ihnen im Fall eines versehentlichen Entfernens das Zertifikat mit Leichtigkeit wiederherzustellen und es an seinen ursprünglichen Ort zurück zu verschieben. Sollte ein Zertifikat 30 Tage im Papierkorb verweilen wird es automatisch endgültig gelöscht, sofern man es vorher nicht manuell im Papierkorb endgültig gelöscht hat. Ein Zertifikat, welches in der App gelöscht worden ist, lässt sich trotzdem neu einscannen. Sollte es sich noch im Papierkorb befinden und daher nicht endgültig entfernt worden sein erhält man einen Hinweis, dass es wiederhergestellt wurde.",
+                            "<b>iOS:</b><ul><li>Wischen Sie unter Status (vor Version 2.12 'Startseite') nach unten und wählen Sie unter 'Mehr' den 'Papierkorb' aus.</li><li>Sollten sich Zertifikate in Ihrem Papierkorb befinden, können Sie ein Zertifikat antippen, um es wiederherzustellen.</li><li>Um ein Zertifikat endgültig aus dem Papierkorb zu entfernen tippen Sie zunächst oben rechts auf 'Bearbeiten'. Anschließend haben Sie die Möglichkeit entweder auf das rote Minuszeichen neben dem jeweiligen Zertifikat zu tippen, oder während des Gedrückthaltens des jeweiligen Zertifikats nach links zu wischen.</ul>",
+                            "<b>Android:</b><ul><li>Tippen Sie unter Status (vor Version 2.12 'Startseite') auf das Drei-Punkte-Menü oben rechts in der Ecke.</li><li>Tippen Sie daraufhin auf 'Papierkorb'.</li><li>Tippen Sie innerhalb eines der dort aufgelisteten Zertifikate auf das Drei-Punkte-Menü oben rechts.</li><li>Wählen Sie aus, ob Sie das ausgewählte Zertifikat wiederherstellen oder endgültig aus dem Papierkorb entfernen wollen.</li></ul>",
+                            "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots für das Warnen in Vertretung für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Papierkorb</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Papierkorb</a></li></ul>"

```

Für das Warnen in Vertretung?

```suggestion
                            "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots zum Papierkorb für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Papierkorb</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Papierkorb</a></li></ul>"
```

---

Forgot to change that while copying from another part of the FAQ, thanks!